test/WebJobs.Script.Tests/Configuration/ActiveHostConfigurationSourceTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Azure.WebJobs.Script.Extensions;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
+{
+    public class ActiveHostConfigurationSourceTests
```

Comment:
Nice tests! One more suggestion for these -- create a configuration with two InMemory dictionaries registered -- where the second one overrides values in the first, and ensure that it retrieves the correct one.

That would specifically be checking the code in `Load()` where you check for `ContainsKey`.